### PR TITLE
Updating changelog, package.json and npm-shrinkwrap for 1.4.15

### PR DESCRIPTION
**Pull request by moizjv**
@imurchie @jlipps @Jonahss please review!

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "appium",
-  "version": "1.4.14",
+  "version": "1.4.15",
   "npm-shrinkwrap-version": "5.4.0",
-  "node-version": "v0.12.7",
```
Comment (Jonahss) [inline npm-shrinkwrap.json]: node-version changed?

**Comment by Jonahss** [comment]
If we're going to update all these versions of modules, we should run the full test suite, not just the stuff related to our change.

**Comment by imurchie** [comment]
I tend to agree.

**Comment by moizjv** [comment]
@Jonahss @imurchie not changing anything in shrinkwrap except the appium version

**Comment by Jonahss** [comment]
yeah thats looks cool

**Comment by imurchie** [comment]
LGTM

**Pull request merged**